### PR TITLE
Set a reason for closing a job

### PR DESCRIPTION
**Pull request by siapankina**
ticket : 
https://www.pivotaltracker.com/story/show/178315029

added radio button with 2 options : Job Completed and No Access, so when job is being closed they can choose a reason 
Selecting a reason is mandatory 

Screenshot 2021-06-14 at 09 57 47

### CODE REVIEW

File: src/utils/hact/work-order-complete/close-job.test.js [hunk 1]
```diff
@@ -15,15 +15,74 @@ describe('buildCloseWorkOrderData', () => {
...
+    }
+
+    const response = buildCloseWorkOrderData(
+      completionDate,
+      notes,
+      reference,
+      'Job Completed'
+    )
+    expect(response).toEqual(CloseWorkOrderFormData)
+  })
```
Comment (NeilvB) [inline src/utils/hact/work-order-complete/close-job.test.js:60]: Do we need this test? It seems be a duplicate of the one above..

File: src/utils/hact/work-order-complete/close-job.test.js [hunk 2]
```diff
@@ -15,15 +15,74 @@ describe('buildCloseWorkOrderData', () => {
...
+      workOrderReference: {
+        id: reference,
+        description: '',
+        allocatedBy: '',
+      },
+      jobStatusUpdates: [
+        {
+          typeCode: '70',
+          otherType: 'complete',
+          comments: `Work order closed - ${notes}`,
```
Comment (NeilvB) [inline src/utils/hact/work-order-complete/close-job.test.js:74]: Do we need to also include some text indicating the selected option? The card seems to suggest we'd need `Work Order Closed - No Access` as a prefix.

File: src/utils/hact/work-order-complete/close-job.test.js [hunk 3]
```diff
@@ -15,15 +15,74 @@ describe('buildCloseWorkOrderData', () => {
...
+      workOrderReference: {
+        id: reference,
+        description: '',
+        allocatedBy: '',
+      },
+      jobStatusUpdates: [
+        {
+          typeCode: '0',
+          otherType: 'complete',
+          comments: `Work order closed - ${notes}`,
```
Comment (NeilvB) [inline src/utils/hact/work-order-complete/close-job.test.js:47]: I think the card requires us to add a prefix like `Work Order Closed - Completed` to this.

**Review by NeilvB — CHANGES_REQUESTED** [review]
Shaping up nicely but I have a couple of queries about the notes.

File: src/utils/hact/work-order-complete/close-job.test.js (hunk 2 above)
Comment (siapankina) [reply src/utils/hact/work-order-complete/close-job.test.js:74]: yes, b/e is going to do that

File: src/utils/hact/work-order-complete/close-job.test.js (hunk 3 above)
Comment (siapankina) [reply src/utils/hact/work-order-complete/close-job.test.js:47]: the same as with No Access 
when I talked with Kyle and Josh they said that b/e will do it 
however, I can ask them about that tomorrow. Maybe will be faster to do it on our side

File: src/utils/hact/work-order-complete/close-job.test.js (hunk 1 above)
Comment (siapankina) [reply src/utils/hact/work-order-complete/close-job.test.js:60]: yup, made a duplication

**Review by NeilvB — APPROVED** [review]
